External/Tools/FDBuild/Program.cs

```diff
@@ -44,10 +44,12 @@ public static int Main(string[] args)
                 catch { }
 
             string swfmillPath = options.SwfmillPath ?? Path.Combine(toolsDir, "swfmill");
-            if (File.Exists(Path.Combine(swfmillPath, "swfmill.exe")))
+            if (File.Exists(Path.Combine(swfmillPath, "swfmill.exe"))) //Windows binary
                 SwfmillLibraryBuilder.ExecutablePath = Path.Combine(swfmillPath, "swfmill.exe");
+            else if (File.Exists(Path.Combine(swfmillPath, "swfmill"))) //Linux binary
```

Comment:
I would add a check for the OS not being Windows. Also, I just saw it's the same for mtascPath v_v. Sorry for not coming up with this before.